plugins/qeta-backend/src/database/DatabaseQetaStore.ts

```diff
@@ -99,6 +99,16 @@ export class DatabaseQetaStore implements QetaStore {
       query.where('questions.author', '=', options.author);
     }
 
+    if (options.searchQuery) {
+      // Only works for postgres but should scale well when used together with the GIN index created in the migration.
+      query.whereRaw(
+        `to_tsvector('english', questions.title || ' ' || questions.content) @@ websearch_to_tsquery(?)`,
```

Comment:
I think this is awesome but we should still support both databases. How about doing this conditionally only for postgres and use the where query for other databases? You can get the database type from `knex.client.config.client` as string